applications/users/tests/test_user_registration.py

```diff
@@ -0,0 +1,82 @@
+from django.core import mail
+from django.test import TestCase
+import factory
+from rest_framework.test import APITestCase, APIClient, APIRequestFactory
+from rest_framework.views import status
+
+from .factories import UserFactory
+from applications.users.models import User
+
+class UserRegistrationTests(APITestCase):
+    url = '/api/v1/auth/registration/'
+    format = 'json'
+
+    @staticmethod
+    def build_params():
+        params = factory.build(dict, FACTORY_CLASS=UserFactory)
+        params['password1'] = params['password2'] = params['password']
+        return params
+
```

Comment:
I think there's an extra line here. Are you using flake8 to check the linting and formatting warning/errors?

Reply:
I'm going to add flake8 in a new PR